src/muya/lib/parser/parse.js

```diff
@@ -13,12 +13,18 @@ delete validateRules['backlash']
 const getSrcAlt = text => {
   const SRC_REG = /src\s*=\s*("|')([^\1]+?)\1/
   const ALT_REG = /alt\s*=\s*("|')([^\1]+?)\1/
+  const WIDTH_REG = /width\s*=\s*("|')([^\1]+?)\1/
+  const HEIGHT_REG = /height\s*=\s*("|')([^\1]+?)\1/
```

Comment:
We should extract only `height` and `width` if it's a number. According to w3schools the attributes unit is `pixels` (unsigned integer).

Reply:
👍I'll update this PR.

Reply:
fixed